python_src/src/lamp_py/performance_manager/l1_rt_trips.py

```diff
@@ -217,6 +217,83 @@ def update_static_version_key(db_manager: DatabaseManager) -> None:
     process_logger.log_complete()
 
 
+def update_start_times(db_manager: DatabaseManager) -> None:
+    """
+    Add in start times to trips that do not have one. For scheduled trips,
+    look these up in the static schedule by finding the earliest Stop Time for
+    that trip id. Added trips cannot be found in the static schedule, so use
+    the earliest vehicle position information we have about them.
+    """
+    missing_start_times = (
+        sa.select(TempEventCompare.pm_trip_id)
+        .distinct()
+        .where(TempEventCompare.start_time.is_(None), TempEventCompare.new_trip)
+        .subquery("missing_start_times_sub")
```

Comment:
I don't think we actually use this `TempEventCompare.new_trip` column in the pipeline. We should probably drop from the schema.